cardano-config/test/Golden/ShelleyGenesis

```diff
@@ -1 +1 @@
-{"activeSlotsCoeff":6.259,"protocolParams":{"poolDecayRate":0,"poolDeposit":0,"protocolVersion":{"minor":0,"major":0},"decentralisationParam":1.9e-2,"maxTxSize":2048,"minFeeA":0,"maxBlockBodySize":239857,"keyMinRefund":0,"minFeeB":0,"eMax":0,"extraEntropy":{"tag":"NeutralNonce"},"maxBlockHeaderSize":217569,"keyDeposit":0,"keyDecayRate":0,"nOpt":100,"rho":0,"poolMinRefund":0,"tau":0,"a0":0},"protocolMagicId":838299499,"startTime":"2009-02-13T23:13:09Z","genDelegs":{"23d51e91ae5adc7ae801e9de4cd54175fb7464ec2680b25686bbb19452b4ed96":"839b047f56e50654bdb504832186dc1ee0c73c8de2daec7ae62738273be825b2"},"updateQuorum":16991,"maxMajorPV":25446,"initialFunds":{"011c14ee8e58fbcbd48dc7367c95a63fd1d937ba989820015db16ac7e5a2e89798e37a65ea2f9bcefb645de4312cf13d8ac12ae61cf242a9aa2973c9ee32e99ce2":12157196},"maxLovelaceSupply":71,"networkMagic":4036000900,"epochLength":1215,"staking":null,"slotsPerKESPeriod":8541,"slotLength":8,"maxKESEvolutions":28899,"securityParam":120842}
\ No newline at end of file
+{"activeSlotsCoeff":6.259,"protocolParams":{"poolDecayRate":0,"poolDeposit":0,"protocolVersion":{"minor":0,"major":0},"minUTxOValue":0,"decentralisationParam":1.9e-2,"maxTxSize":2048,"minFeeA":0,"maxBlockBodySize":239857,"keyMinRefund":0,"minFeeB":0,"eMax":0,"extraEntropy":{"tag":"NeutralNonce"},"maxBlockHeaderSize":217569,"keyDeposit":0,"keyDecayRate":0,"nOpt":100,"rho":0,"poolMinRefund":0,"tau":0,"a0":0},"protocolMagicId":838299499,"startTime":"2009-02-13T23:13:09Z","genDelegs":{"23d51e91ae5adc7ae801e9de4cd54175fb7464ec2680b25686bbb19452b4ed96":"839b047f56e50654bdb504832186dc1ee0c73c8de2daec7ae62738273be825b2"},"updateQuorum":16991,"maxMajorPV":25446,"initialFunds":{"011c14ee8e58fbcbd48dc7367c95a63fd1d937ba989820015db16ac7e5a2e89798e37a65ea2f9bcefb645de4312cf13d8ac12ae61cf242a9aa2973c9ee32e99ce2":12157196},"maxLovelaceSupply":71,"networkMagic":4036000900,"epochLength":1215,"staking":null,"slotsPerKESPeriod":8541,"slotLength":8,"maxKESEvolutions":28899,"securityParam":120842}
```

Comment:
Again, I think if we can show we didn't need to change the golden tests, it keeps us honest with not breaking the existing testnet.

Reply:
`goldenTestJSON` checks that the encoding of `exampleShelleyGenesis` is equal to the content of the golden file and also that the decoding of the golden file is equal to `exampleShelleyGenesis`. Therefore, because the `ToJSON` instance for `PParams` now includes the new `minUTxOValue` field, we need to also update the golden file. 

If you want, we _could_ do a "decode-only" check using `goldenTestJSONDec`. That would only check that the decoded golden file is equal to `exampleShelleyGenesis`.